### ISSUE TITLE
Check install script fails to download hg19.genome, but still creates an empty hg19.genome

### ISSUE THREAD

**Issue by hdashnow** [MelbourneGenomics/cpipe#3]
When the check install script fails to download hg19.genome, it still creates an empty file hg19.genome. So if you rerun install.sh it succeeds.

```
$ ./pipeline/scripts/install.sh

...

================================================================
Check if genome file /vlsci/VR0193/shared/cpipe/hg19.genome exists ...
================================================================


The chromosome size file (.genome) does not exist. Create it by downloading from UCSC (requires MySQL client) (y/n) y
./pipeline/scripts/install.sh: line 345: mysql: command not found

========================= ERROR ==================================

Failed to download genome file

==================================================================
```

But then, if I rerun the check_install script:

```
================================================================
Check if genome file /vlsci/VR0193/shared/cpipe/hg19.genome exists ...
================================================================


================================================================
Check ulimit ...
================================================================


================================================================
WARNING: The limit on open files is set to 1024. Cpipe may require more open files than
this. Consider adding 'ulimit set -S -n 2048' to your .bashrc file.
================================================================


================================================================
Success: all the dependencies I know how to check are OK
================================================================
```

However hg19.genome is empty:

```
$ ls -l /vlsci/VR0193/shared/cpipe/hg19.genome
-rw-rw-r-- 1 hdashnow VR0193 0 Apr 23 11:32 /vlsci/VR0193/shared/cpipe/hg19.genome
```

Perhaps some logic to prevent the hg19.genome file from being created if the command fails, or clean up incomplete download?

**Comment by hdashnow** [comment]
Also, what is hg19.genome? And where do I download it manually?

I see from config.groovy that "Genome needed (currently) only if using expanded splice regions"

Yet the install script says:
`WARNING: Cpipe will not operate correctly unless the /vlsci/VR0193/shared/cpipe/hg19/hg19.genome file is created`

Might want to add some clarifying statements to the install script, and the documention (it's not currently mentioned in the documentation PDF or the hg19 README)

**Comment by hdashnow** [comment]
Fixed by #5 
Will open a separate issue for the .genome file
